Gemfile.lock

```diff
@@ -103,7 +103,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
It looks like you either need this username/password *or* you need to reset the Gemfile/lockfile before committing.

More info here.